### PR TITLE
Feature/update doc

### PR DESCRIPTION
**Pull request by nkvuong**
- Upgrade AWS provider for AWS guides
- Add examples for account-level identities

### CODE REVIEW

File: docs/resources/service_principal.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ subcategory: "Security"
 ---
 # databricks_service_principal Resource
 
-Directly manage [Service Principals](https://docs.databricks.com/administration-guide/users-groups/service-principals.html) that could be added to [databricks_group](group.md) within workspace.
+Directly manage [Service Principals](https://docs.databricks.com/administration-guide/users-groups/service-principals.html) that could be added to [databricks_group](group.md) in Databricks workspace or account.
+
+To create service principals in the Databricks account, the provider must be configured with `host = "https://accounts.azuredatabricks.net"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and `auth_type  = "azure-cli"` on Azure deployments
```
Comment (nfx) [inline docs/resources/service_principal.md:8]: is it really available for User Principals only? Is it the same SPN OBO flow issue? :(

Comment (nkvuong) [reply docs/resources/service_principal.md]: 🤦  - corrected it to AAD tokens

File: docs/resources/group.md [hunk 2]
```diff
@@ -3,7 +3,9 @@ subcategory: "Security"
...
 
-This resource allows you to manage [groups in Databricks Workspace](https://docs.databricks.com/administration-guide/users-groups/groups.html) or [Account Console](https://accounts.cloud.databricks.com/) (for AWS deployments). You can also [associate](group_member.md) Databricks users and [service principals](service_principal.md) to groups. This is useful if you are using an application to sync users & groups with SCIM API.
+This resource allows you to manage [groups in Databricks Workspace](https://docs.databricks.com/administration-guide/users-groups/groups.html), [Databricks Account Console](https://accounts.cloud.databricks.com/) or [Azure Databricks Account Console](https://accounts.azuredatabricks.net). You can also [associate](group_member.md) Databricks users and [service principals](service_principal.md) to groups. This is useful if you are using an application to sync users & groups with SCIM API.
+
+To create groups in the Databricks account, the provider must be configured with `host = "https://accounts.azuredatabricks.net"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using AAD tokens on Azure deployments
```
Comment (nfx) [inline docs/resources/group.md:8]: Azure Databricks in both cases? :-)

File: docs/resources/group_member.md [hunk 3]
```diff
@@ -3,7 +3,9 @@ subcategory: "Security"
 ---
 # databricks_group_member Resource
 
-This resource allows you to attach [users](user.md) and [groups](group.md) as group members.
+This resource allows you to attach [users](user.md), [service_principal](service_principal.md) and [groups](group.md) as group members.
```
Comment (nfx) [inline docs/resources/group_member.md:6]:
```suggestion
This resource allows you to attach [users](user.md), [service_principal](service_principal.md), and [groups](group.md) as group members.
```

File: docs/resources/group_member.md [hunk 4]
```diff
@@ -3,7 +3,9 @@ subcategory: "Security"
...
+
+To attach members to groups in the Databricks account, the provider must be configured with `host = "https://accounts.azuredatabricks.net"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using AAD tokens on Azure deployments
```
Comment (nfx) [inline docs/resources/group_member.md:8]: Azure Databricks in both cases?

File: docs/resources/service_principal.md [hunk 5]
```diff
@@ -3,7 +3,9 @@ subcategory: "Security"
 ---
 # databricks_service_principal Resource
 
-Directly manage [Service Principals](https://docs.databricks.com/administration-guide/users-groups/service-principals.html) that could be added to [databricks_group](group.md) within workspace.
+Directly manage [Service Principals](https://docs.databricks.com/administration-guide/users-groups/service-principals.html) that could be added to [databricks_group](group.md) in Databricks workspace or account.
+
+To create service principals in the Databricks account, the provider must be configured with `host = "https://accounts.azuredatabricks.net"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using AAD tokens on Azure deployments
```
Comment (nfx) [inline docs/resources/service_principal.md:8]: Both azure?

File: docs/resources/user.md [hunk 6]
```diff
@@ -3,7 +3,9 @@ subcategory: "Security"
...
+This resource allows you to manage [users in Databricks Workspace](https://docs.databricks.com/administration-guide/users-groups/users.html), [Databricks Account Console](https://accounts.cloud.databricks.com/) or [Azure Databricks Account Console](https://accounts.azuredatabricks.net). You can also [associate](group_member.md) Databricks users to [databricks_group](group.md). Upon user creation the user will receive a password reset email. You can also get information about caller identity using [databricks_current_user](../data-sources/current_user.md) data source.
+
+To create users in the Databricks account, the provider must be configured with `host = "https://accounts.azuredatabricks.net"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and authenticate using AAD tokens on Azure deployments
```
Comment (nfx) [inline docs/resources/user.md:8]: Both azure?)

**Review by nfx — CHANGES_REQUESTED** [review]
Few last changes required

File: docs/resources/service_principal.md (hunk 5 above)
Comment (nkvuong) [reply docs/resources/service_principal.md:8]: ctrl+c, ctrl+v error

**Review by nfx — APPROVED** [review]
Lgtm

**Pull request merged**